### DISCUSSION TITLE
Minimum settings_example.py possible and license

### DISCUSSION THREAD

**Discussion by armandomiani** [WeblateOrg/weblate#6462] [Q&A]
Hello guys, 

I'm writing an open-source deployment for Weblate and facing a dilemma. 

As far as I'm concerned, if I reuse the whole https://github.com/WeblateOrg/weblate/blob/main/weblate/settings_example.py in another open-source repository maybe I'm violating the Weblate GPL's license. 

Then I'd like to strip the example, use a minimum settings file and refer to the original one if the user wants to learn more configurations. 

As we're talking about a ~1000 lines of configuration code, it's hard to know whether a setting is required or not. 

Do you guys have some example or documentation telling me what's the required settings? And also, using the settings_example would incur in GPL violations? 

Thanks!

**Comment by nijel** [comment]
You might want to use https://github.com/WeblateOrg/weblate/blob/main/weblate/settings_docker.py and configure Weblate via environment variables in a same way as Docker container is configured (see https://docs.weblate.org/en/latest/admin/install/docker.html). You can use it by setting environment DJANGO_SETTINGS_MODULE=weblate.settings_docker before starting Weblate.
